dataactbroker/handlers/fileHandler.py

```diff
@@ -642,3 +727,17 @@ def getProtectedFiles(self):
 
         response["urls"] = self.s3manager.getFileUrls(bucket_name=CONFIG_BROKER["static_files_bucket"], path=CONFIG_BROKER["help_files_path"])
         return JsonResponse.create(StatusCode.OK, response)
+
+    def completeGeneration(self, generationId):
+        """ Retrieve the generated file, using the specified ID to look up submission and file type """
```

Comment:
Not essential, but I guarantee we're going to have no idea what `generationId` and "generated file" mean when we look at this in a few months. Is there something more illuminating that we could note in this docstring? Maybe something like
> For files D1 and D2, we generate a "FileGenerationTask" with a unique identifier "generationId". A third party then posts data to us using that identifier. Handle that post by writing the relevant D file

?